From 2801ae9b65fff13b917c8c56befdd75d47fe7c2f Mon Sep 17 00:00:00 2001
From: Adam YH Lee <adam.yh.lee@gmail.com>
Date: Mon, 5 Jan 2015 17:45:49 -0800
Subject: [PATCH 06/15] ARM:dts Improve U-Boot and DTS compatibility for
 LCD43/35

OMAP DSS still uses Kernel boot arguments to set and enable the default
video out (omapdss.def_disp). Historically it is set to lcd43 for the
4.3in LCD screen, and lcd35 for the 3.5in LCD screen. By setting the device
label of the display node in the DTS to match this U-Boot parameter, user
does not need to make the change himself.

Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 arch/arm/boot/dts/omap3-overo-common-lcd35.dtsi | 2 +-
 arch/arm/boot/dts/omap3-overo-common-lcd43.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/omap3-overo-common-lcd35.dtsi b/arch/arm/boot/dts/omap3-overo-common-lcd35.dtsi
index 233c69e..df8908a 100644
--- a/arch/arm/boot/dts/omap3-overo-common-lcd35.dtsi
+++ b/arch/arm/boot/dts/omap3-overo-common-lcd35.dtsi
@@ -120,7 +120,7 @@
 
 	lcd0: display@0 {
 		compatible = "lgphilips,lb035q02";
-		label = "lcd";
+		label = "lcd35";
 
 		reg = <1>;					/* CS1 */
 		spi-max-frequency = <10000000>;
diff --git a/arch/arm/boot/dts/omap3-overo-common-lcd43.dtsi b/arch/arm/boot/dts/omap3-overo-common-lcd43.dtsi
index 7e5d786..606dfe8 100644
--- a/arch/arm/boot/dts/omap3-overo-common-lcd43.dtsi
+++ b/arch/arm/boot/dts/omap3-overo-common-lcd43.dtsi
@@ -98,7 +98,7 @@
 
 	lcd0: display@0 {
 		compatible = "samsung,lte430wq-f0c", "panel-dpi";
-		label = "lcd";
+		label = "lcd43";
 
 		pinctrl-names = "default";
 		pinctrl-0 = <&lte430_pins>;
-- 
1.9.1

